src/components/ChallengesComponent/ChallengeCard/index.js

```diff
@@ -282,16 +324,17 @@ class ChallengeCard extends React.Component {
 
 ChallengeCard.defaultPrps = {
   shouldShowCurrentPhase: true,
-  showError: () => {},
+  // showError: () => {},
   reloadChallengeList: () => {}
 }
 
 ChallengeCard.propTypes = {
   challenge: PropTypes.object,
   shouldShowCurrentPhase: PropTypes.bool,
-  showError: PropTypes.func,
+  // showError: PropTypes.func,
```

Comment:
@mark-nakachon as `showError` is not in use, it would be better to remove it completely instead of commenting.

Also, as we don't use it anywhere anymore, could you please remove it from the parent component also:

![image](https://user-images.githubusercontent.com/146016/104178671-6628e500-5413-11eb-80b8-7dff162a807e.png)